packages/overlay/stories/overlay.stories.ts

```diff
@@ -1413,3 +1413,76 @@ export const virtualElement = (args: Properties): TemplateResult => {
 virtualElement.args = {
     placement: 'right-start' as Placement,
 };
+
+export const virtualElementDeclaratively = (
+    args: Properties
+): TemplateResult => {
+    const virtualTrigger = new VirtualTrigger(0, 0);
+    const handleContextmenu = async (event: PointerEvent): Promise<void> => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        const overlay = document.querySelector(
+            'sp-overlay:not([open])'
+        ) as Overlay;
+        virtualTrigger.updateBoundingClientRect(event.clientX, event.clientY);
+        overlay.willPreventClose = true;
+        overlay.open = true;
+    };
+    const overlay = (): TemplateResult => html`
+        <sp-overlay
+            offset="0"
+            type="auto"
+            placement=${args.placement}
+            .triggerElement=${virtualTrigger}
```

Comment:
Binding these both to the same virtual trigger does a weird thing in which always both menus are in the same place making it look like the previously opened Menu winks closed without making its transition. I think maybe we make a `virtualTrigger` for each and then address them in `handleContextmenu` as `overlay.triggerElement.updatedBindingClientRect()` instead.

Reply:
I updated it with separate virtual triggers... However I don't see any difference in transition :(

Reply:
When comparing https://65812823fcc49166eb0173e8--spectrum-web-components.netlify.app/storybook/?path=/story/overlay--virtual-element-declaratively to https://chore-overlay-declarative-context-menu--spectrum-web-components.netlify.app/storybook/?path=/story/overlay--virtual-element-declaratively you're not able to see the current minus one Menu fade out? If not, check them out at 10% speed:

Previous (one virtual trigger):

https://github.com/adobe/spectrum-web-components/assets/1156657/dbd51a4d-4f1f-47f6-82eb-0848876df0d3

Current (two virtual triggers):


https://github.com/adobe/spectrum-web-components/assets/1156657/21b869d3-36c9-4cbc-971f-3ffc89a6a6f5

Reply:
ok damn! It wasn't even noticeable for me. Thanx for attaching the video!